### DISCUSSION TITLE
写的代码也太难懂了

### DISCUSSION THREAD

**Discussion by aa0604** [w7corp/easywechat#2690] [General]
我很感谢你做出的这几个版本的贡献，但是我想吐槽一下。你的4.x的代码基本上没办法看，感觉只有你自己才能看得懂。比如我外网想获取component_verify_ticket，想保存下来到内网做开发，但却发现不知道怎么才能获取到，我能搜到位置和方法，却不知道如何调用……而看代码也难以看懂无法跟踪。你这是只能自己玩，等你自己一段时间没维护，估计也会忘了玩不动。没有文档就动不了的程序。最后你的6.x就告诉大家，简单封闭了底层逻辑，大家拿接口路径+参数就能用了。事实上，如果你的代码写得好，你一个1.x最多2.x的版本就能既能封装好业务逻辑，又好用易懂了。因为架构好的话，要更新，新开发，继承之前版本的方法逻辑是很容易的

**Comment by overtrue** [comment]
多谢指教🙏

**Comment by reatang** [comment]
确实，做php就应该像楼主说的一样。写好业务就行了🙏

**Comment by tuzhiltd** [comment]
所以该怎么调用方法获取“ component_verify_ticket”？
